src/utils/classNames.js

```diff
@@ -3,6 +3,10 @@
     <div className={classNames(['base-class', {'second' : someFlag, 'third' : true}])}></div>
 */
 const classNames = (names) => {
+    if (!names) {
```

Comment:
nit: @marcinbodnar makes sense to add test if you find some bug with edge case (given this already has tests)